app/instance-initializers/violations-helper.js

```diff
@@ -0,0 +1,11 @@
+import ViolationsHelper from 'ember-a11y-testing/utils/violations-helper';
+
+export function initialize(application) {
+  window.violationsHelper = new ViolationsHelper();
```

Comment:
What if we did something like [`ember-devtools`](https://github.com/aexmachina/ember-devtools#globalize) did and only set it to a global if you want to?